src/application.rs

```diff
@@ -200,6 +205,10 @@ impl<'a> Application<'a> {
                     self.state.windows.remove(&endpoint);
                 }
             }
+            NetMessage::StreamAudio(audio) => match audio {
+                Some(audio) => self.state.pulse_audio(audio),
```

Comment:
I'm curious if this handles already multi user stream, or do i need to tag each users stream (live video stream).

@lemunozm Are you able to test this? currently I don't a second linux pc.

Basicly the question is what happens when different audio sources are merged, do the audio remain understandable?

Reply:
I don't have another Linux either 😢.
Seeing how it works I suppose that the audio is mixed, but I do not see it as a problem.
The problem could be with sound in two directions. I think it will be coupled.

Maybe it could be cool that it behaves more like a WhatsApp audio in the sense of if I send an audio, the receiver could choose when to listen to it. Even it can listen to the same audio several times. This feature would imply navigating along the conversation allowing to select an audio to listen to.

Reply:
The whatsup idea is cool, it could be the same for sending pictures and files.

It could be for now simply different commands, stream-audio/stream-video | send-audio/send-video, Currently we're finishing the first set, the second set should be possible after some ui work.